11-model-comparison/11_model_comparison.ipynb

```diff
@@ -0,0 +1,462 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    )
</pre><p>Consider explaining the purpose of these lines</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>